fjord/feedback/jinja2/feedback/android_feedback.html

```diff
@@ -0,0 +1,192 @@
+                      {% endtrans %}
+                    </div><!-- .message -->
+                    <div>
+                      {% trans %}
+                        It takes less than a minute and feels great.
+                      {% endtrans %}
+                    </div>
+                    <a class="stars" id="play-store" data-event="FeedbackOpenPlay" href="#">
+                      <svg>
+                        <defs>
+                          <symbol viewBox="0 0 100 100" id="star">
+                            <polygon points="48 75 18.6107374 90.4508497 24.2235871 57.7254249 0.447174185 34.5491503 33.3053687 29.7745751 48 0 62.6946313 29.7745751 95.5528258 34.5491503 71.7764129 57.7254249 77.3892626 90.4508497" class="star" xmlns="http://www.w3.org/2000/svg" fill="#0092db"></polygon>
+                          </symbol>
+                        </defs>
+                        <use width="25" height="25" xlink:href="#star" x="0" y="0"></use>
+                        <use width="25" height="25" xlink:href="#star" x="25" y="0"></use>
+                        <use width="25" height="25" xlink:href="#star" x="50" y="0"></use>
+                        <use width="25" height="25" xlink:href="#star" x="75" y="0"></use>
+                        <use width="25" height="25" xlink:href="#star" x="100" y="0"></use>
+                      </svg>
+                      {% trans %}
+                        Yes, go to Google Play
+                      {% endtrans %}
+                    </a>
+                  </div><!-- .message-box -->
+                </div><!-- .happy -->
+
+                <div class="sad">
+                  <h1>
+                    {% trans %}
+                      Oh no!
+                    {% endtrans %}
+                  </h1>
+
+                  <div class="ask">
+                    <form id="responseform" action="" method="post">
+                      <label for="description">
+                        {% trans %}
+                          We're sorry that you had some problems with Firefox.
+                          Please tell us what happened so that we can fix it.
+                        {% endtrans %}
+                      </label>
+                      <div class="input-box">
+                        <div id="description-counter">{{ TRUNCATE_LENGTH }}</div>
+                        <textarea data-max-length="{{ TRUNCATE_LENGTH }}" id="description" name="description"></textarea>
+                      </div><!-- .input-box -->
+
+                      <label for="id_url">
+                        {% trans %}
+                          If your feedback is related to a website, you can include it here:
+                        {% endtrans %}
+                      </label>
+                      <div class="input-box">
+                        <input class="url" id="id_url" name="url" placeholder="http://" type="text">
+                      </div><!-- .input-box -->
```

Comment:
I'm pretty sure this is different than the existing feedback form in Fennec. In Fennec, there's a checkbox that says "Include last visited site" and lists the last visited site.

We'd need Fennec to pass that along somehow because only they know what it would be.